lib/tasks/unsubscribe.rake

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+
+namespace :unsubscribe do
+  def unsubscribe!(email_address)
```

Comment:
In risk of starting to sound like @kevindew, I don't think this should be a `!` method.

Reply:
Righto, removed :)